### DISCUSSION TITLE
MY GANACHE SOFTWARE CAN'T SHOW THE MAIN TAB (TAB THAT SHOWS ADDRESSES)(MAC)

### DISCUSSION THREAD

**Discussion by Egorov050** [ConsenSys-archive/truffle#5840] [General Q&A]
When i push QUICK START, this tab opens, but i want the normal one. I ve uninstalled it many times but nothing happens , still the same(MAC)
<img width="1440" alt="Screenshot 2023-01-16 at 13 28 53" src="https://user-images.githubusercontent.com/115251460/212656799-83b70b81-c31a-41f4-81ea-833b18c4ed6e.png">
pls, help me(

**Comment by FlaviuWeb3** [comment]
Hi @Egorov050 could you please let me know if you're looking for the following view?

<img width="1197" alt="Screenshot 2023-01-19 at 16 53 42" src="https://user-images.githubusercontent.com/97533870/213474546-b41494ca-968f-477f-b029-d571d67d2066.png">

**Reply by Egorov050** [reply]
yep

**Reply by FlaviuWeb3** [reply]
Please hit the "Restart" button from top corner right, and you should be able to see the above view.
